public/index.html

```diff
@@ -7,11 +7,13 @@
     <link rel="icon" type="image/png" sizes="32x32" href="/icons/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="16x16" href="/icons/favicon-16x16.png">
     <link rel="manifest" href="/icons/site.webmanifest">
-    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#333333">
+    <link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#9b1b01">
     <link rel="shortcut icon" href="/icons/favicon.ico">
-    <meta name="msapplication-TileColor" content="#333333">
+    <meta name="apple-mobile-web-app-title" content="HedgeDoc">
+    <meta name="application-name" content="HedgeDoc">
+    <meta name="msapplication-TileColor" content="#e8a85a">
     <meta name="msapplication-config" content="/icons/browserconfig.xml">
-    <meta name="theme-color" content="#ffffff">
+    <meta name="theme-color" content="#e8a85a">
```

Comment:
This color is not part of the official theme colors.

Reply:
Yes, that's right. I forgot to change the index.html.